doc/JSON_INFO.md

```diff
@@ -376,22 +376,34 @@ Mods can modify this via "add:traits" and "remove:traits".
 "description": "Nothing gets you down!" // In-game description
 "starting_trait": true, // Can be selected at character creation (default: false)
 "valid": false,      // Can be mutated ingame (default: true)
+"purifiable": false, //Cannot be purified (default: true)
```

Comment:
Misleading description: the "purifiable" member defines whether the mutation can be purified, *not* whether it cannot be purified. Compare to the line above.